api/v1alpha1/syncconfig_types.go

```diff
@@ -49,7 +49,7 @@ type SyncConfigStatus struct {
 	// Important: Run "make" to regenerate code after modifying this file
 
 	// ReconcileError contains any error messages during synchronization, if any
-	ReconcileError string `json:"reconcileerror,omitempty"`
+	ReconcileError string `json:"reconcileError,omitempty"`
```

Comment:
Is it possible to update this on an existing CRD?
Or do we need to introduce a new version (i.e. `v1alpha2`)?

Reply:
I was able to update the CRD on the lab cluster and then downgrade again to the old CRD.
But that was a while ago, I will do a retest

Reply:
To be clear: This field is new and came with the Operator-SDK Migration to 1.0.0, which is nowhere rolled out on our clusters. Since the status spec is currently only for human and also only added after a reconciliation, I don't expect problems when we update our CRD definition with new status fields, it shouldn't break the SyncConfig spec itself.